dissect/target/helpers/fsutil.py

```diff
@@ -512,10 +512,12 @@ def open_decompress(
         An binary or text IO stream, depending on the mode with which the file was opened.
 
     Example:
-        bytes_buf = open_decompress(Path("/dir/file.gz")).read()
+        .. code-block:: python
 
-        for line in open_decompress(Path("/dir/file.gz"), "rt"):
-            print(line)
+            bytes_buf = open_decompress(Path("/dir/file.gz")).read()
+
+            for line in open_decompress(Path("/dir/file.gz"), "rt"):
+                print(line)
```

Comment:
Wouldn't just `Example::` achieve the same? (with a double `:`)

Reply:
Also works, just renders it a bit differently

(Notice the capitalisation of Example)

Using `..code-block:: python`:

![image](https://github.com/user-attachments/assets/906f29f1-c324-43d0-a330-acd0892cf7e9)

Using `Example::`:

![image](https://github.com/user-attachments/assets/2ff92328-89bb-401a-ab8a-dc270034ba79)

Reply:
The `Example::` does have the benefit that you don't have to indent it twice. Let me know what is preferred :)

Reply:
I prefer the first, because that capitalisation means that Sphinx has properly picked it up as an example block.

Reply:
I know, but I guess that's a bug in Sphinx?